owl-zentangle-1.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="description" content="Browse Caitlin's abstract art">
+    <meta name="keywords" content="zentangles, abstract, art, drawing">
+
+    <title>Caitlin's Art</title>
+
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="container">
+      <div class="grey-background">
+        <a name="top"></a>
+        <a href="index.html"><img src="images/personal-logo-transparent.png" class="logo"></a>
+        <div class="navigation">
+          <ul>
+            <li><a href="about.html">about</a></li>
+            <li><a href="art.html">art</a></li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="primary-image-content">
+        <img src="images/owl-zentangle-1.jpg" class="primary-image greyscale">
+        <h2>Owl Zentangle</h2>
+        <p>This is a custom piece of art. You will be receiving a mailed copy of the original artwork with signature, not a printed copy.</p>
+        <p>Cost: $_____</p>
+        <p>Please contact me for details if you are interested.</p>
+        <div class="buy-now">
+          <a href="mailto:caitlin.daitch@gmail.com">Buy Now</a>
+        </div>
+      </div>
+
+      <div class="shop top-line">
+        <h2>You may also like:</h2>
+        <a href="#"><img src="http://placehold.it/300x300" class="art"></a>
+        <a href="#"><img src="http://placehold.it/300x300" class="art"></a>
+        <a href="#"><img src="http://placehold.it/300x300" class="art"></a>
+      </div>
+
+      <div class="footer grey-background">
+        <a href="index.html"><img src="images/personal-logo-transparent.png" class="footer-logo"></a>
+        <a href="#top"><img src="images/return-to-top.png" class="footer-return"></a>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
Overall keep up the good work - start filling in some of the details and you'll be really close to being done. Great job on the responsive stuff. If you have any questions about my comments let me know but when you're ready go ahead and hit the big green "Merge" button at the bottom of the Pull Request.